### ISSUE TITLE
remove dropout option from CuDNNLSTM layers

### ISSUE THREAD

**Issue by mr-francois** [hiddengenome/deepG#57]
The author of keras, Francois Chollet, suggests that using regular dropout in stacked RNNs will "wreck training" (https://github.com/keras-team/keras/issues/8935)

**Comment by philippmuench** [comment]
is the Concrete Dropout implementation from here what we want?
https://blogs.rstudio.com/tensorflow/posts/2018-11-12-uncertainty_estimates_dropout/
